addons/common/stringtable.xml

```diff
@@ -68,105 +68,139 @@
         </Key>
         <Key ID="STR_ZEN_Common_Unchanged">
             <English>Unchanged</English>
+            <Russian>Не изменилось</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_VeryLow">
             <English>Very Low</English>
+            <Russian>Оч. низкий</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Low">
             <English>Low</English>
+            <Russian>Низкий</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Medium">
             <English>Medium</English>
+            <Russian>Средний</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_High">
             <English>High</English>
+            <Russian>Высокий</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_VeryHigh">
             <English>Very High</English>
+            <Russian>Оч. высокий</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_VeryWeak">
             <English>Very Weak</English>
+            <Russian>Оч. слабый</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Weak">
             <English>Weak</English>
+            <Russian>Слабый</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Strong">
             <English>Strong</English>
+            <Russian>Сильный</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Extreme">
             <English>Extreme</English>
+            <Russian>Экстремальный</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_All">
             <English>All</English>
+            <Russian>Все</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Side">
             <English>Side</English>
+            <Russian>Сторона</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Faction">
             <English>Faction</English>
+            <Russian>Фракция</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Aircraft">
             <English>Aircraft</English>
+            <Russian>Самолёт</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Filter">
             <English>Filter</English>
+            <Russian>Фильтр</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Units">
             <English>Units</English>
+            <Russian>Солдаты</Russian>
```

Comment:
It's not only soldiers, could be just some unit (inc. civilian side)

Reply:
that depends on particular point of application. we have quite a bunch of synonyms, which can be applied under that word.